lib/router.js

```diff
@@ -52,15 +55,18 @@ module.exports = (config) => {
     path: paths.translations + '/__lng__/__ns__.json'
   });
 
-  router.use(hof.middleware.deepTranslate({
+  app.set('views', [paths.templates].concat(config.sharedViews));
+
+  app.use(expressPartialTemplates(app));
+
+  app.use(hof.middleware.deepTranslate({
     translate: i18n.translate.bind(i18n)
   }));
 
-  router.use(mixins(fields));
+  app.use(mixins(fields));
 
   const wizardConfig = {
-    controller: config.baseController || hof.controllers.base,
-    templatePath: paths.templates
+    controller: config.baseController || hof.controllers.base
```

Comment:
As we set baseController to hof.controller.base in defaults.js, this could be `controller: config.baseController`